### ISSUE TITLE
Latest n diagonal in heatmap()

### ISSUE THREAD

**Issue by kennethshsu** [casact/chainladder-python#278]

**Comment by kennethshsu** [comment]
Ahh, similar solution to what I came up with.

```python
raa = cl.load_sample("raa")
cl.Development().fit_transform(raa[raa.valuation.year >= raa.valuation_date.year-3]).link_ratio.heatmap()
```

I think I am getting better at this. Only problem is now we can't see the older link ratios.😅

**Comment by jbogaardt** [comment]
Cool. By your approach you don't even need the `Development` estimator:
```python
raa = cl.load_sample('raa')
raa.link_ratio[raa.link_ratio.valuation.year >= raa.valuation_date.year-3].heatmap()
```

**Comment by kennethshsu** [comment]
Yes, even better!

**Comment by jbogaardt** [comment]
Should this be reclassifed discussion forum Q&A?

**Comment by kennethshsu** [comment]
Yes, unless you think it's useful to add an argument to only heatmap the last n diagonals? It's a bit better as we can see the older diagonals that are not heat-mapped. But it will mean that it's another piece of code that we have to maintain. What do you think?

**Comment by jbogaardt** [comment]
I fear n_diagonals will be short lived. Eventually someone will want the same but with drop_hi and drop_low. I dont want to create more args for those use cases. 

Maybe it's better to create a Boolean arg in heatmap that allows the dropped link ratios from the Development estimator to be visible or not.

**Comment by kennethshsu** [comment]
That's actually a great idea. I like that.

With this we should be able to see if the remaining LDFs have any patterns.
